### ISSUE TITLE
Plot History device expand/collapse button keyboard focus is hardly identifiable in HC themes

### ISSUE THREAD

**Issue by AlexanderSher** [Microsoft/RTVS#3381]
1. Open R Tools > Plots > Plot History Window
2. Switch to R Plot history pane using Ctrl + Tab
3. Using Shift + Alt navigate between pane contents and toolbar
4. Using down arrow key navigate to Device expand/collapse group controls and Observe the keyboard focus in HC black

**Actual Result:**
Expand/collapse group control when focused using keyboard is not identifiable

**Expected Result:**
Expand/collapse group control when focused using keyboard should be identifiable
